### DISCUSSION TITLE
ImageUploader iOS15.6.1 上传内存特别大的图不触发onChange事件

### DISCUSSION THREAD

**Discussion by AspirinMrmi** [ant-design/ant-design-mobile#5660] [Q&A]
ImageUploader在iOS15.6.1上传内存特别大的图片（大于30MB）时不会触发input的onChange事件，有没有什么hack方案？在同事的iOS13.6版本上是没问题的。

**Comment by awmleer** [comment]
和 #5661 重复了，这个 lock 掉了
